pkg/controller/gslb/internal/depresolver/depresolver_test.go

```diff
@@ -0,0 +1,241 @@
+package depresolver
```

Comment:
will fix makefile targets.

GH actions are ok, For testing it takes lintpath as root (`/pkg/controller`), otherwise linting/testing will not pass on generated functions  like `schema_pkg_apis_ohmyglb_v1beta1_GslbStatus`. 

Here is message from build log

```
GOLINTPATH=pkg/controller
...
...
go test ./...
...
...
github.com/AbsaOSS/ohmyglb/cmd/manager	[no test files]
github.com/AbsaOSS/ohmyglb/pkg/apis	[no test files]
github.com/AbsaOSS/ohmyglb/pkg/apis/ohmyglb	[no test files]
github.com/AbsaOSS/ohmyglb/pkg/apis/ohmyglb/v1beta1	[no test files]
github.com/AbsaOSS/ohmyglb/pkg/controller	[no test files]
github.com/AbsaOSS/ohmyglb/pkg/controller/gslb	0.286s
github.com/AbsaOSS/ohmyglb/pkg/controller/gslb/internal/depresolver	0.028s
github.com/AbsaOSS/ohmyglb/pkg/controller/gslb/internal/utils	[no test files]
github.com/AbsaOSS/ohmyglb/version	[no test files]

```

Reply:
btw: maybe we can enter to operator SDK and rewrite generator to provide linter safe names